core/runner/src/test/java/com/cognifide/aet/runner/distribution/dispatch/CollectorJobSchedulerTest.java

```diff
@@ -96,15 +96,19 @@ public void add_expectMessageSent() throws Exception {
   }
 
 
-  @Ignore("Ignored for some unexpected test failure on Linux")
-  @Test(timeout = 2000l, expected = IllegalStateException.class)
-  public void add_whenAddedMessageExists_expectIllegalStateException() throws Exception {
+  @Test(expected = IllegalStateException.class)
```

Comment:
Now I can see that the thing we assert here is that an exception is thrown. However, it's still unclear to me which method all is expected to throw an exception and why. 

There are better ways to unit-test exceptions than specifying the `expected` parameter of the `@Test` annotation.

You can be a bit more expressive if you use the [`ExpectedException` rule](https://github.com/junit-team/junit4/wiki/exception-testing#expectedexception-rule). 

Personally, I find this library useful when testing exceptions: https://github.com/Codearte/catch-exception